### ISSUE TITLE
How to execute this ?

### ISSUE THREAD

**Issue by TheNamesNeo** [rist0/Fortnite-Checker#1]
dude from where do i launch it ? its a sln format i cant find exe format !!!

**Comment by LilDipper-TaterTots** [comment]
amen to that, how would a man start this

**Comment by jackahr** [comment]
same issue no clue how to run the program

**Comment by w-koperski** [comment]
You need to compile this.

**Comment by Phsiic** [comment]
> You need to compile this.

How do I do that? I'm not very good with coding and stuff like that :P

**Comment by KrazyD83** [comment]
has anyone got this working ?? Cant find the .exe anywhere .... Can someone please help me so i dont have to go through 100 addresses 1 at a time ???

**Comment by LilDipper-TaterTots** [comment]
Yeah, it's older so I doubt the O.W. will swoop in and I have not been able
to compile

**Comment by ghost** [comment]
1. Head to https://visualstudio.microsoft.com/ and download the Visual Studio installer (Community is free).

2. When prompted, select the packages you would like to install (Include the C# packages at the top of the window) then go through the installation process.

3. When Visual Studio is set up, go to the Fortnite Checker folder. Go into the src folder and open the FortniteChecker.sln file. This is the project file and it will show all the programs in the Visual Studio window.

3.5. If you would like to change any code, you can find and edit the programs in the Solution Explorer window on the right.

4. Click on the green start button at the top of the window. This will build the exe file and run it.

To run this after building it with Visual Studio (without having to run it through VS), just search for the exe file in the Fortnite Checker folder. (Search for the bin folder). In this folder is the exe file. Its type is “Application”. Click on that and it will run.

**Comment by DefNotEbbi** [comment]
You need Visual Studio installed specifically with .net framework (version 4.6.1). Download these two and run the code.
